database/exip.sql

```diff
@@ -214,6 +214,7 @@ CREATE TABLE `Buyer` (
   `canContactBuyer` tinyint(1) DEFAULT NULL,
   `exporterIsConnectedWithBuyer` tinyint(1) DEFAULT NULL,
   `exporterHasTradedWithBuyer` tinyint(1) DEFAULT NULL,
+  `connectionWithBuyerDescription` varchar(1000) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
```

Comment:
Not a blocker - thinking about splitting up this table for any new fields we have in No PDF (need a think/hopefully new PR)